source/admin_manual/list_of_events.rst

```diff
@@ -854,579 +482,384 @@ These events apply only for connections using the ``connection API``.
-+---------------------+------------------------------------------------------+
-| remote_pid          | Remote UNIX socket server's process ID.              |
-+---------------------+------------------------------------------------------+
-| remote_uid          | Remote UNIX socket server's system user ID.          |
-+---------------------+------------------------------------------------------+
-
-client_connection_connected
----------------------------
-
-Emitted when a server accepts an incoming client connection.
-
-
-client_connection_disconnected
-------------------------------
-
-Emitted when a client connection is terminated.
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| bytes_in            | Amount of data read, in bytes                        |
-+---------------------+------------------------------------------------------+
-| bytes_out           | Amount of data written, in bytes                     |
-+---------------------+------------------------------------------------------+
-| reason              | Disconnection reason                                 |
-+---------------------+------------------------------------------------------+
-
-
-server_connection_connected
----------------------------
-
-Emitted when an outgoing server connection was either successfully established
-or failed. Currently it is not possible to know which one happened.
-
-server_connection_disconnected
-------------------------------
-
-Emitted when a server connection is terminated.
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| bytes_in            | Amount of data read, in bytes                        |
-+---------------------+------------------------------------------------------+
-| bytes_out           | Amount of data written, in bytes                     |
-+---------------------+------------------------------------------------------+
-| reason              | Disconnection reason                                 |
-+---------------------+------------------------------------------------------+
 
 FS
 ==
 
-.. _event_fs:
+.. dovecot_core:event:: fs
 
-fs
---
+   Inherits from environment (e.g. :mail_user event)
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from environment (e.g. :ref:`event_mail_user`)                    |
-+---------------------+------------------------------------------------------+
+   .. todo:: WHAT DOES THIS MEAN???
```

Comment:
Means it could be inherited from all kinds of different things. I guess it could be said differently, like `May be inherited from various different parents (e.g. :mail_user event) or even from no parent.`